src/gui/EditWidget.ui

```diff
@@ -66,7 +66,7 @@
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
-        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
+        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
```

Comment:
@droidmonkey we could also preserve the same order for the `Ok` `Apply` and `Cancel` buttons here.